### ISSUE TITLE
LaTeX rendering

### ISSUE THREAD

**Issue by asher-pembroke** [danielfrg/mkdocs-jupyter#12]
I'm having trouble rendering equations with this plugin.

Neither of these work for me:

```latex
\begin{equation} f(x) = y \end{equation}
$$ g(f) = g(f(x)) $$
```

With `mknotebooks` I usually have to add the mathjax cdn 

```yaml
extra_javascript: 
     - https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML
```
I've also tried swapping this cdn with the one mentioned in #11 but no dice.

**Comment by Sayam753** [comment]
Hi @asher-pembroke 
After updating the `mkdocs-jupyter` to 0.13.0, I reverted back to use 0.12.0 because there's no LaTeX script [here](https://github.com/danielfrg/mkdocs-jupyter/blob/master/mkdocs_jupyter/templates.py#L7-L8) in 0.13.0 and rendering failed.

**Comment by vincentsarago** [comment]
FYI, I also struggle with this. Locally I was able to visualize my jupyter notebook in Mkdocs but on the CI (building web docs) it was failing with 
```
jinja2.exceptions.TemplateNotFound: mathjax.tpl
```

reverting to `mkdocs-jupyter==0.12` as mentioned by @Sayam753 fixed this error 🙏

**Comment by Sayam753** [comment]
Yes @vincentsarago 
I once updated nbconvert to `6.0.0` and then CSS for rendering of cells got messed up. So, I reverted back to use `nbconvert==5.6.1`.
One more thing, rendering of LaTeX equations works fine with `mkdocs-jupyter==0.13.0` and `mkdocs-material==5.5.0`. For `mkdocs-material>5.5.0`, the larger LaTeX equations do not get wrapped up and are not responsive as well. I do not know why.

**Comment by MSeal** [comment]
The underlying nbconvert issues should be resolved for this issue. However the template setup looks like it might be slightly incorrect. https://github.com/danielfrg/mkdocs-jupyter/blob/master/mkdocs_jupyter/templates/mkdocs_html/conf.json#L2 should probably be basing off lab since you extend lab settings, which in turn will pull base settings as well. Doing this you should change the [dependency extension in the template](https://github.com/danielfrg/mkdocs-jupyter/blob/master/mkdocs_jupyter/templates/mkdocs_html/notebook.html.j2#L3) to just `index.html.j2` to match.

**Comment by danielfrg** [comment]
Thanks for the info and catching the template setup error MSeal.

I just fixed that and released a new version that pins it to `nbconvert>=6.0.7` and released it on pypi `mkdocs-jupyter==0.15.1`.

On my tests everything works, please create a new issue if this still a problem.
